docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -0,0 +1,273 @@
+                    "settings": {
+                        "minDockerVersion": "v1.25",
+                        "registryCredentials": {
+                            "tsmregistry": {
+                                "address": "10.16.7.4",
+                                "password": "$$$0meCoMPL3xP4$$W0rd001!@#$$",
+                                "username": "nestedcrimagepulltoken"
+                            }
+                        }
+                    },
+                    "type": "docker"
+                },
+                "schemaVersion": "1.1",
+                "systemModules": {
+                    "edgeAgent": {
+                        "settings": {
+                            "image": "10.16.7.4/azureiotedge-agent:1.0",
+                            "createOptions": ""
+                        },
+                        "type": "docker",
+                        "env": {
+                            "SendRuntimeQualityTelemetry": {
+                                "value": "false"
+                            }
+                        }
+                    },
+                    "edgeHub": {
+                        "settings": {
+                            "image": "10.16.7.4/azureiotedge-hub:1.0",
+                            "createOptions": "{\"HostConfig\":{\"PortBindings\":{\"443/tcp\":[{\"HostPort\":\"443\"}],\"5671/tcp\":[{\"HostPort\":\"5671\"}],\"8883/tcp\":[{\"HostPort\":\"8883\"}]}}}"
+                        },
+                        "type": "docker",
+                        "status": "running",
+                        "restartPolicy": "always"
+                    }
+                }
+            }
+        },
+        "$edgeHub": {
+            "properties.desired": {
+                "routes": {
+                    "route": "FROM /messages/* INTO $upstream"
+                },
+                "schemaVersion": "1.1",
+                "storeAndForwardConfiguration": {
+                    "timeToLiveSecs": 7200
+                }
+            }
+        }
+    }
+}
+```
+
+Use the information from the previous sections to update the relevant JSON values.
+
+You will use the file path in the next section when you run the command to apply the configuration to your device.
+
+## Deploy the connected registry module on IoT Edge
+
+Use the following command to deploy the connected registry module on the IoT Edge device:
+
+```azurecli
+az iot edge set-modules \
+  --device-id [device id] \
+  --hub-name [hub name] \
+  --content [file path]
+```
+
+For more details you can refer to the [Deploy Azure IoT Edge modules with Azure CLI](../iot-edge/how-to-deploy-modules-cli.md) article.
+
+To check the status of the connected registry, use the following CLI command:
+
+```azurecli
+az acr connected-registry show \
+  --registry mycontainerregistry001 \
+  --name myconnectedmirror \
+  --output table
+```
+
+You may need to a wait few minutes until the deployment of the connected registry completes.
```

Comment:
@toddysm the user can confirm that their connected registry is connected to the cloud when they run `az acr connected-registry show` and see `connectionState: Online`

Reply:
Added the response from the command